cookbooks/plume_2D_chunk/doc/plume.tex

```diff
@@ -0,0 +1,90 @@
+
+\begin{figure}
+  \centering
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel3}\\
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T3}\\
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta3}
+  \caption{\it Plume in a 2D chunk: Columns from left to right: isoviscous case, weakly temperature dependent case, and strongly temperature-dependent case. Rows from top to bottom: Velocity, temperature, and viscosity field at steady state. Angular opening of $\pi/8$.}
+  \label{fig:plume-diff-creep}
+\end{figure}
+
+Obtaining a steady state is contingent on the narrow angular opening. We find that simply increasing the angular opening from $\pi/8$ to $\pi/4$ yields only a statistical steady state, as multiple downwellings occur near the side but the system never stabilises (see Fig.~\ref{fig:plume-angular-opening}). Also, decreasing $\eta_0$ by a factor 10 would yield $Ra=10^7$. In this case, too, a statistical steady state is reached (not shown here).
+
+\begin{figure}
+  \centering
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/exp1_22}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/exp1_45}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/exp1_90}
+  \caption{\it Plume in a 2D chunk: Temperature at the end of the run. From left to right: angular opening of 
+$\pi/8$, $\pi/4$ and $\pi/2$. The first two have reached a steady state while the third one has not.}
+  \label{fig:plume-angular-opening}
+\end{figure}
+
+Fig.~\ref{fig:plume-diff-creep-vrms} shows the time evolution of the root mean square velocity as a function of time. 
+As mentioned above, no active planet is at steady state so the time on the horizontal axis is not really meaningful. Also, it is easy to show that the path to steady state (if at all attained) is vastly influenced by the initial temperature field.
+Since viscosities increase on average in the domain with an increase in activation energy $E$, we find that the corresponding average velocities are then smaller. Also, although it looks like the system reaches steady state for the $\pi/2$ opening angle it ultimately proves unstable and becomes chaotic, akin to the figure on the cover of this manual. Please check the corresponding video \url{https://youtu.be/FN2BBmbiA8E} to see the system for the entire duration of the simulation. 
```

Comment:
In the sentence that starts with "since viscosities", what does "then smaller" refer to? Maybe say "we find that...velocities decrease with increasing activation energy"?